reformat general inplace_increment #2813

edaehn opened the pull request:
+examples, array, container update

juliagsy commented on ivy/array/general.py:
```diff
@@ -1038,24 +1038,38 @@ def inplace_update(
 
     def inplace_increment(
-        self: ivy.Array, val: Union[ivy.Array, ivy.NativeArray]
+        self: Union[ivy.Array, ivy.NativeArray], val: Union[ivy.Array, ivy.NativeArray]
```
The original type hints for `self` (`ivy.Array`) is correct as this is an instance method for ivy.Array, so the type must be ivy.Array only

juliagsy reviewed (changes requested):
Hello! Great work so far! One tiny change and you're good to go! As usual, please request my review once again when you're done! Thank you!

edaehn commented:
Hi @juliagsy, thanks for your message. I have fixed the type. I needed to [reopen the PR](https://github.com/unifyai/ivy/pull/4159), and fixed the type. Thanks for your advice. Could you please check it up?
